src/components/templates/PartyMap/components/Map/Map.tsx

```diff
@@ -216,48 +263,66 @@ export const Map: React.FC<PropsType> = ({
     () =>
       venue.rooms.map((room) => (
         <PartyMapRoomOverlay
+          // TODO: is room.title unique? Is there something better we can use for the key?
           key={room.title}
           venue={venue}
           room={room}
           attendances={attendances}
-          setSelectedRoom={setSelectedRoom}
-          setIsRoomModalOpen={setIsRoomModalOpen}
-          // onEnterRoom={enterPartyMapRoom}
+          enterRoom={enterPartyMapRoom}
+          selectRoom={selectRoom}
+          unselectRoom={unselectRoom}
+          // // onEnterRoom={enterPartyMapRoom}
         />
       )),
-    [attendances, setIsRoomModalOpen, setSelectedRoom, venue]
+    [attendances, enterPartyMapRoom, selectRoom, unselectRoom, venue]
   );
+
   if (!user || !venue) {
     return <>Loading map...</>;
   }
 
+  // TODO: this was the old way we were adding the map background, now we're using img below. Is that what we want?
+  // <div
+  //     className="camp-grid-container"
+  //     style={{
+  //       backgroundImage: `url(${venue.mapBackgroundImageUrl})`,
+  //       backgroundSize: "cover",
+  //       display: "grid",
+  //       gridTemplateColumns: `repeat(${templateColumns}, calc(100% / ${templateColumns}))`,
+  //       gridTemplateRows: `repeat(${templateRows}, 1fr)`,
+  //     }}
+  // >
+
   return (
     <div className="party-map-content-container">
       <div className="party-map-container">
         <div className="party-map-content">
+          {/* TODO: do we want to use img here? see above code snippet from the old Camp Map template */}
           <img
             width="100%"
             className="party-map-background"
             src={venue.mapBackgroundImageUrl}
             alt=""
           />
```

Comment:
Chris: I think on the `Camp` this may have been done so we can auto-calculate the number of rows.

Glenn: as a CSS `backgroundImage` we also don't get ricght-click "save image" (I think), so could be better to keep the CSS pattern?

Reply:
Chris: there was a specific commit on the `Camp` `Map` component about why we switched to using CSS. I don't think there is any explicit reason we switched to an `img` tag on the `PartyMap`